### ISSUE TITLE
HELP - Can't get past Grub

### ISSUE THREAD

**Issue by tmcd35** [holoiso-eol/holoiso#569]
Foolishly just run latest updates from gui.  As per known issue re: yay-git the deck UI no longer boots.  Did [CTRL]-[ALT]-[F3] and followed instruction to rollback vulkan driver.  Now the system won't boot past the grub menu.  Which ever option I choose I get a series of zstd buffer errors and a stack overflow error.  Press any key takes me back to the grub menu, and so it loops.

If I boot into a live USB image is there anything I can do to recover my installation?

Thanks.

**Comment by oblende** [comment]
Possibly. After booting the USB open open the icon that will chroot to your Holoiso installation. From there type grub-install at the command line to reinstall the grub bootloader. This should probably fix the issue.

**Comment by tmcd35** [comment]
Thank you. Took a bit more wirk but ultimately grub-install kicked the system back to life after I deleted /etc/default/grub
I was able to get back the terminal and reinstall the correct previous vulkan driver.

I now have a new issue. Steam games that ran fine previously now refuse to start.  Linux native emulators (Dolphin, RPCS3, RetroArch) all work fine, as do most really old non steam games I added that use Proton. But all Steam games are borked.
Tried deleting a prefix file but that didn't work. Sonething between Proton and the Graphics Driver has been thrown off alignment.

Anyway, marking this as completed. Problem solved. New issue can wait until after the coming update (see if that fixes it).
